opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -792,6 +809,14 @@ export const meEditField = async (context, user, userId, inputs, password = null
   if (user.external && PROTECTED_EXTERNAL_ATTRIBUTES.includes(key)) {
     throw ForbiddenAccess();
   }
+  // Check the user has the capability to edit some fields
+  const capabilities = user.capabilities.map((c) => c.name);
+  if (
+    !capabilities.some((c) => USER_EDITION_CAPABILITIES.includes(c))
```

Comment:
I don't think we should test capabilities, because this API meEdit should not be used to change other fields than the one listed, for all users.

Reply:
Maybe it's easier sometimes for an admin that wants to edit itself to do it via meUser